### PR TITLE
Ryt tmp

### PR DESCRIPTION
**Pull request by declmal**
Thanks for contributing to TVM!   Please refer to guideline https://docs.tvm.ai/contribute/ for useful information and tips. After the pull request is submitted, please request code reviews from [Reviewers](https://github.com/dmlc/tvm/blob/master/CONTRIBUTORS.md#reviewers).

### CODE REVIEW

File: cvm/quantization/tfm_pass.py [hunk 1]
```diff
@@ -290,31 +288,42 @@ def _impl(op, params, graph, **kwargs):
 
     return th_dict
 
-def requant_operator(X, oname, def_prec, oscale=None, **kwargs):
-    logger = logging.getLogger('log.mrt.simulate')
+def realize(X, sb, prec, **kwargs):
+    name = kwargs.get('name', N.n(X.attr('name')))
```
Comment (wlt-cortex) [inline cvm/quantization/tfm_pass.py]: The kwargs only contains `name` params, why not define the argument `name` with a default `NONE` value?

File: cvm/quantization/tfm_pass.py [hunk 2]
```diff
@@ -290,31 +288,42 @@ def _impl(op, params, graph, **kwargs):
...
+    if sb == 0:
+        sym = mx.sym.Custom(X, precision=prec,
+                cvm_name=name, name=name, op_type='cvm_clip')
```
Comment (wlt-cortex) [inline cvm/quantization/tfm_pass.py]: The cvm_name is not needed in fact, try to remove it and remove the definition source code in `cvm_op.py`.

File: cvm/quantization/transformer.py [hunk 3]
```diff
@@ -65,14 +70,23 @@ def quantize(self, no_realize=False):
...
+        self._qext = self._get_ext()
+        return self._qsym, self._qprm, self._qext
+
+    def dump(self, fname="test_dump", directory="~/tvm-cvm/data/"):
+        assert self._qsym is not None
+        import os
+        directory = os.path.expanduser(directory)
+        qsym_path = os.path.join(directory, fname+'.json')
+        qprm_path = os.path.join(directory, fname+'.params')
+        qext_path = os.path.join(directory, fname+'.ext')
```
Comment (wlt-cortex) [inline cvm/quantization/transformer.py]: use the `extend_fname` function in `utils.py`

**Review by wlt-cortex — CHANGES_REQUESTED** [review]

File: cvm/quantization/tfm_pass.py (hunk 2 above)
Comment (declmal) [reply cvm/quantization/tfm_pass.py]: fixed

File: cvm/quantization/transformer.py (hunk 3 above)
Comment (declmal) [reply cvm/quantization/transformer.py]: fixed

File: cvm/quantization/tfm_pass.py (hunk 1 above)
Comment (declmal) [reply cvm/quantization/tfm_pass.py]: fixed

**Pull request merged**